### ISSUE TITLE
Screen reader does not announce sub actions on list items

### ISSUE THREAD

**Issue by misolori** [microsoft/vscode#125547]
Steps to Reproduce:

1. Enable screen reader
2. Open command palette
3. Navigate to item that has a gear for configuring 
4. :bug: Screen reader does not announce that there is sub actions (gear)

I would expect that screen readers would announce "Use tab to see options..." or something like that. I didn't get that on macOS using the voice over. The same also applies to any of our list actions with sub actions (SCM, etc.)

Version: 1.57.0-insider
Commit: fe1eafc80c07608de52a2f9c99fee103dea77f5e
Date: 2021-06-04T12:19:25.391Z
Electron: 12.0.9
Chrome: 89.0.4389.128
Node.js: 14.16.0
V8: 8.9.255.25-electron.0
OS: Darwin x64 20.5.0

**Comment by isidorn** [comment]
Thanks for creating this issue. 
We usually do not announce if sub actions are present. So if we were to change this, we should probably do it for all list elements that contain actions. In practice I believe that users press `tab` to explore if there are sub-actions.
I will bring this up in the Gitter accessibility channel to get some user feedback.

**Comment by misolori** [comment]
Yes, it would be good to get feedback from others if this is a pattern done elsewhere. Someone reached out and asked how users know if there are any actions there.
